Polygon_mesh_processing/test/Polygon_mesh_processing/test_detect_features.cpp

```diff
@@ -0,0 +1,48 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
in the test you should add calls to each documented function, with and without named parameters, so that we make sure they are all used (and in particular the default parameters)